Jenkinsfile

```diff
@@ -68,12 +78,13 @@ node {
                 withCredentials([
                         usernamePassword(credentialsId: 'sonatype', passwordVariable: 'sonatypePassword', usernameVariable: 'sonatypeUsername'),
                         string(credentialsId: 'gpgpassphrase', variable: 'gpgPassPhrase')]) {
-                    sh "${gradleCommand} publish " +
-                            " -Ppublish_username=${sonatypeUsername} " +
-                            " -Ppublish_password=${sonatypePassword} " +
-                            " -PkeyId=DF8285F0 " +
-                            " -Ppassword=${gpgPassPhrase} " +
-                            " -PsecretKeyRingFile=/var/jenkins_home/secring.gpg "
+                    Publish(sonatypeUsername, sonatypePassword, gpgPassPhrase, "apix-interface")
```

Comment:
Eh, you know you can ensure that you only apply the `maven-publish` plugin on projects that you want to publish, right?
Or at least run all those tasks in one gradle invocation.